### PR TITLE
Add satooshi/php-coveralls to require dev

### LINKED ISSUE #2: coveralls.io hook not working
@sandrokeil can you take a look why coveralls is not working? I've set everything up and travis-ci works/is green but coveralls status is unknown.

### PR DESCRIPTION
**Pull request by codeliner**
Let's see if it fixes #2

### CODE REVIEW

**Comment by codeliner** [comment]
@sandrokeil Can you take a look? Error #2 still exists. Not sure, how to solve it.

**Comment by sandrokeil** [comment]
@codeliner This is not the solution. We have to remove the [src_dir](https://github.com/prooph/psr7-middleware/blob/master/.coveralls.yml#L3) defintion. Seems it's [not necessary anymore](https://github.com/satooshi/php-coveralls#coverallsyml).

File: composer.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
   "require-dev": {
     "phpunit/phpunit": "^4.8 || ^5.0",
     "fabpot/php-cs-fixer": "^1.11",
+    "satooshi/php-coveralls": "dev-master",
```
Comment (prolic) [inline composer.json]: 1.0 is finally released, so we should update to use ^1.0, same goes for all other prooph-components

**Comment by codeliner** [comment]
ok, but the same config works for our other components. Do we need to add the dependency? It is missing in current composer.json

**Comment by prolic** [comment]
src_dir is not necessary, right. See travis on php 5.6:

```
[Symfony\Component\Config\Definition\Exception\InvalidConfigurationException]  

Unrecognized option "src_dir" under "coveralls"
```

Coveralls has changed this with 1.0 release (only a few days old).

**Comment by prolic** [comment]
My question, why is coveralls only executed on 5.6?

**Comment by sandrokeil** [comment]
@prolic We need only one analysis, because there is no difference between the PHP versions. coveralls is loaded in the [travis.yml](https://github.com/prooph/psr7-middleware/blob/master/.travis.yml#L27). However, we should pin it to version 1.0 there.

**Comment by prolic** [comment]
Another thing:

.travis.yml should be like this:

after_success:
- travis_retry php vendor/bin/coveralls -v

we use after_script, which is always executed.

**Comment by codeliner** [comment]
everything set, let's see if it works

**Comment by codeliner** [comment]
new error:
[Symfony\Component\Config\Definition\Exception\InvalidConfigurationExcept  
  ion]  
    coverage_clover XML file is not readable

**Comment by codeliner** [comment]
I'm going to merge this PR. @sandrokeil can you try to fix it with a new PR then? I need to finish today.

**Comment by sandrokeil** [comment]
@codeliner I will check it

**Comment by prolic** [comment]
build/logs/clover.xml is wrong in https://github.com/prooph/psr7-middleware/blob/master/composer.json#L70

**Comment by codeliner** [comment]
thx

**Comment by sandrokeil** [comment]
@prolic My fault, i will fix it. Or you will do. ;-)

**Pull request merged**